### PR TITLE
Release Nov 9, 2023

### PR DESCRIPTION
**Pull request by artpark**
### Summary

The following packages will be released:

  - terra-collapsible-menu-view@6.84.0
  - terra-data-grid@1.1.0
  - terra-date-picker@4.93.0
  - terra-date-time-picker@4.99.0
  - terra-dialog-modal@3.95.0
  - terra-embedded-content-consumer@3.40.0
  - terra-file-path@1.0.1
  - terra-form-validation@1.90.0
  - @cerner/terra-framework-docs@1.44.0
  - terra-menu@6.80.0
  - terra-slide-panel-manager@5.84.0
  - terra-slider@1.1.0
  - terra-table@5.2.0
  - terra-tabs@7.11.2

### Testing

This change was tested using:

- [ ] WDIO
- [ ] Jest
- [ ] Visual testing (please attach a screenshot or recording)
- [ ] Other (please describe below)
- [x] No tests are needed

### Reviews

In addition to engineering reviews, this PR needs:

- [ ] UX review
- [ ] Accessibility review
- [ ] Functional review

### Additional Details

---

Thank you for contributing to Terra.
@cerner/terra

### CODE REVIEW

**Review by sdadn — APPROVED** [review]

File: packages/terra-data-grid/CHANGELOG.md [hunk 1]
```diff
@@ -2,6 +2,8 @@
 
 ## Unreleased
 
+## 1.1.0 - (November 9, 2023)
+
 * Fixed
   * Fixed the column header background color for the Orion Fusion theme.
   * Fixed pinned column divider issue during keyboard resize.
```
Comment (nikhitasharma) [inline packages/terra-data-grid/CHANGELOG.md:9]: Should these packages with fixes be a patch release instead of a minor release?

Comment (artpark) [reply packages/terra-data-grid/CHANGELOG.md:9]: There is some stuff added as well like the ability to select a range of cells to FlowsheetDataGrid, so I will keep this a minor version

File: packages/terra-tabs/CHANGELOG.md [hunk 2]
```diff
@@ -2,6 +2,8 @@
 
 ## Unreleased
 
+## 7.12.0 - (November 9, 2023)
```
[the remark this answers is not in the document]
Comment (artpark) [reply packages/terra-tabs/CHANGELOG.md:5]: This one should be a patch version 😅 my bad

File: packages/terra-table/CHANGELOG.md [hunk 3]
```diff
@@ -2,9 +2,12 @@
 
 ## Unreleased
 
+## 5.2.0 - (November 9, 2023)
```
Comment (sdadn) [inline packages/terra-table/CHANGELOG.md:5]: I think we're still doing an alpha release. Can you correct this to `5.1.1-alpha.1`?

**Review by sdadn — APPROVED** [review]

**Pull request merged**